tests/PerformanceHarness/performance_test_basic.py

```diff
@@ -339,6 +339,7 @@ def launchCluster(self):
         return self.cluster.launch(
             pnodes=self.clusterConfig._pNodes,
             totalNodes=self.clusterConfig._totalNodes,
+            totalProducers=self.clusterConfig._pNodes if self.clusterConfig._totalNodes > 21 else None,
```

Comment:
Why is it `None` if `_totalNodes <= 21`?

Reply:
None is the default. It only needs to be set to the non-default if more then 21 total nodes are specified. When more than 21 total nodes are specified is tries to create 22 accounts and fails. The whole pnode, unstartedNodes, totalNodes, prodCount, totalProducers, sharedProducers is a hacked up mess. It really needs to be fixed at some point. It is not clear to me what they all even mean.